src/DynamicRupture/FrictionLaws/BaseFrictionLaw.h

```diff
@@ -34,17 +36,31 @@ class BaseFrictionLaw : public FrictionSolver {
 #endif
     for (unsigned ltsFace = 0; ltsFace < layerData.getNumberOfCells(); ++ltsFace) {
       alignas(ALIGNMENT) FaultStresses faultStresses{};
+      SCOREP_USER_REGION_BEGIN(
+          myRegionHandle, "computeDynamicRupturePrecomputeStress", SCOREP_USER_REGION_TYPE_COMMON)
+      LIKWID_MARKER_START("PrecomputeStress");
       common::precomputeStressFromQInterpolated(faultStresses,
                                                 impAndEta[ltsFace],
                                                 qInterpolatedPlus[ltsFace],
                                                 qInterpolatedMinus[ltsFace]);
+      LIKWID_MARKER_STOP("PrecomputeStress");
+      SCOREP_USER_REGION_END(myRegionHandle)
 
+      SCOREP_USER_REGION_BEGIN(
+          myRegionHandle, "computeDynamicRupturePreHook", SCOREP_USER_REGION_TYPE_COMMON)
+      LIKWID_MARKER_START("PreHook");
```

Comment:
why don't you use the same names ;-)
It is not a bit of overkill to have the profiling instructions for both LIKWID and Scorep?